### PR TITLE
Use HTML5 details tag in bug report issue template [skip ci]

### PR DESCRIPTION
**Pull request by trxcllnt**
Use an HTML5 details tag in the bug report issue template so the output of `print_env.sh` is collapsed by default.

* See a demo at https://gist.github.com/trxcllnt/4843c15a483de805e7b39db740014271
* Read more about this tag at https://gist.github.com/ericclemmons/b146fe5da72ca1f706b2ef72a20ac39d

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -20,8 +20,10 @@ A clear and concise description of what you expected to happen.
  - Environment location: [Bare-metal, Docker, Cloud(specify cloud provider)]
  - Method of cuDF install: [conda, Docker, or from source]
    - If method of install is [Docker], provide `docker pull` & `docker run` commands used
- - Please run and attach the output of the `cudf/print_env.sh` script to gather relevant environment details
- 
+
+<details><summary>Click here to see environment details</summary><pre>
+Please run and attach the output of the `cudf/print_env.sh` script to gather relevant environment details
```
Comment (jrhemstad) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Just to be excruciatingly clear
```suggestion
Please run the `cudf/print_env.sh` script to gather relevant environment details and paste the output inside the `<details> </details>` tags.
```

**Comment by jrhemstad** [comment]
@trxcllnt just need to add a CHANGELOG.md entry.

**Comment by kkraus14** [comment]
@rapidsai/ops Would someone from your team review? Unsure if you want to standardize across the repos.

**Comment by harrism** [comment]
Could the script just be modified to output the <details> tags directly?

**Comment by jrhemstad** [comment]
> Could the script just be modified to output the

That's a great idea.

**Comment by mike-wendt** [comment]
> > Could the script just be modified to output the
> 
> That's a great idea.

@jrhemstad are we going to modify the script as a part of the PR to include @harrism's idea?

**Comment by mike-wendt** [comment]
Let me know what you guys decide with the output of the script. I think we should standardize it and also update the `CONTRIBUTING.md` as well as docs.rapids.ai to mention this process when submitting issues. Happy to do a final review when you guys are ready @kkraus14 @jrhemstad

**Comment by jrhemstad** [comment]
@trxcllnt would you mind updating the `print_env.sh` script to automatically add the `<detail>` tags to the start/end? That should make it easier for copy/paste into issues.

**Comment by trxcllnt** [comment]
@jrhemstad done! I updated the script to indent the output between the HTML tags, otherwise there's a chance GFM won't nest wrap the content in the \<details/\> tags when it renders.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -20,10 +20,7 @@ A clear and concise description of what you expected to happen.
  - Environment location: [Bare-metal, Docker, Cloud(specify cloud provider)]
  - Method of cuDF install: [conda, Docker, or from source]
    - If method of install is [Docker], provide `docker pull` & `docker run` commands used
-
-<details><summary>Click here to see environment details</summary><pre>
-Please run the `cudf/print_env.sh` script to gather relevant environment details and paste the output inside the `<details> </details>` tags.
-</pre></details>
+   - Please run and attach the output of the `cudf/print_env.sh` script to gather relevant environment details
```
Comment (harrism) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Should it be "paste"? or "attach"?  I suppose it doesn't really matter?

File: print_env.sh [hunk 3]
```diff
@@ -29,19 +30,19 @@ lscpu
 echo
 
 echo "***CMake***"
-which cmake && cmake --version 
+which cmake && echo cmake --version
```
Comment (harrism) [inline print_env.sh]: What's with the `echo`? That will just cause it to print the command, rather than the output of the command, won't it?

Comment (trxcllnt) [reply print_env.sh]: @harrism not when I ran it locally ([result here](https://gist.github.com/trxcllnt/4843c15a483de805e7b39db740014271)). The issue I saw was `which cmake` prints a newline after the path, but `read` only sees one a single statement, and doesn't indent the output of `cmake --version`. The `echo` on the second statement gets read as a new line in the indent loop at the bottom.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (trxcllnt) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I prefer paste myself, but I reverted to the previous copy to minimize the PR surface area.

File: print_env.sh (hunk 3 above)
Comment (harrism) [reply print_env.sh]:
```
mharris@dgx14:~/gitlab-nvidia/RAPIDS/cudf/cpp/build$ which cmake && echo cmake --version
/usr/local/bin/cmake
cmake --version
```

Comment (trxcllnt) [reply print_env.sh]: Haha whoops I was looking at the wrong window. Fixed now!

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (mike-wendt) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I think paste works

Also, I believe your indenting is +2 more spaces than necessary

Comment (trxcllnt) [reply .github/ISSUE_TEMPLATE/bug_report.md]: @mike-wendt ah so this is tough -- if I only indent 4 spaces and they paste the `<details>` tag on the same line as the second markdown bullet point, it won't be parsed as details and will look like junk. But if they paste it on a new line, it'll be doubly-nested (with 2 horizontal scrollbars), but at least still collapsed.

Comment (trxcllnt) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I updated this gist with examples of the different indentations people might copy/paste: https://gist.github.com/trxcllnt/4843c15a483de805e7b39db740014271

**Comment by harrism** [comment]
@mike-wendt do you approve?

**Comment by harrism** [comment]
rerun tests

**Comment by kkraus14** [comment]
@harrism [skip ci] prevents CI from running but requires an admin merge. Want me to merge?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (mike-wendt) [reply .github/ISSUE_TEMPLATE/bug_report.md]: @trxcllnt this wasn't about the pasting of the `<details>` tag but how it renders in the issue, as you can see here: https://github.com/rapidsai/cudf/blob/ec4d57d39a29fd5baa14f5526dc434c3e23e9a5a/.github/ISSUE_TEMPLATE/bug_report.md

`Please run ...` is a step and should be at the top level of the list to match the other requested actions

Comment (mike-wendt) [reply .github/ISSUE_TEMPLATE/bug_report.md]: The way it is now, it is nested and I believe it impacts the readability

Comment (mike-wendt) [reply .github/ISSUE_TEMPLATE/bug_report.md]: At this point I think we should break this off into it's own section so it is clearly visible and we don't miss the other pieces of info requested above accidentally

**Comment by mike-wendt** [comment]
@harrism @kkraus14 can you approve or comment on my changes to address the issue that wasn't resolved. Not sure if my approach makes it better or worse

**Pull request merged**